README.md

```diff
@@ -52,6 +58,7 @@ run KOS or Kubernetes components.
 | kapi    |   1    |   4  |   20   |
 | kctrl   |   1    |   2  |   20   |
 | netcd   |   3    |   4  |   20   |
+| netcdOp |   1    |   2  |   20   |
```

Comment:
I was planning to include this on the KOS centralized controllers node.

Reply:
I have no objection to that.